runner-config.md

```diff
@@ -0,0 +1,44 @@
+To run the tests with a customized configuration, you can add a new setting with key named: `java.test.config`.
+
+> [How to create new settings in VS Code?](https://code.visualstudio.com/docs/getstarted/settings#_creating-user-and-workspace-settings)
```

Comment:
It would be great if we can provide extra instruction to guide users to put the configuration in the right place. In most cases, users may want to save the test config in `workspace settings`. So the instruction would be:

> It is recommended to save the test configuration as workspace settings. To open workspace settings, run command `Preference: Open Workspace Settings`. And click on the icon `{}` on the top right to show the source code of your settings.